Chatify/chat/static/js/chat.js

```diff
@@ -5,55 +5,70 @@ app.config(function($interpolateProvider) {
     $interpolateProvider.endSymbol(']}');
 });
 
-
 app.controller('chatCtrl', function($scope, $http) {
+    var ws = new WebSocket('ws://127.0.0.1:8000/ws/chat/')
+
+    ws.onopen = function() {
+        console.log("websocket connection open")
+    }
+
+    $scope.removeOfflineUser = function(chat) {
+         $scope.chatData = $scope.chatData.filter(data => data.id !== chat.id);
+        $scope.$apply()
```

Comment:
Why you are not removing

Reply:
and one more thing if i  write single line function it's not working here with angularJs.

Reply:
![Screenshot from 2023-05-17 13-39-03](https://github.com/Dhruvi-Makwana/Chatify/assets/120730093/d4c563ff-d7fe-4d25-9f06-c757461b5f21)

Reply:
don't write if